setup.py

```diff
@@ -36,6 +36,7 @@ def read(*rnames):
         'Products.statusmessages',
         'Zope2',  # Globals, OFS(tests),
         'decorator',
+        'plone.app.contentlisting',
```

Comment:
P.CMFPlone already depends on p.a.contentlisting, but this doesn't cause any problems as far as I have tested.